paddle/fluid/operators/squeeze_op.cc

```diff
@@ -0,0 +1,200 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
`2016 ` -> `2018`

Reply:
@chengduoZH I noticed that it is [2016](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/reshape_op.cc#L1) in most files

Reply:
@kuke This time should be the time for the file to be created.

Reply:
Done